tests/cypress/tests-e2e/legacyParamImport.cy.js

```diff
@@ -411,4 +440,142 @@ describe('Test on legacy param import', () => {
             cy.readStoreValue('state.position.projection.epsgNumber').should('eq', 3857)
         })
     })
+
+    context('Extra Parameter Imports', () => {
+        it('sets the language correctly', () => {
+            cy.goToMapView({ lang: 'it' }, false)
+            cy.readStoreValue('state.i18n.lang').should('eq', 'it')
+        })
```

Comment:
Dito lang is not a legacy parameter and is already tested, we should remove it from here to keep the number of `it()` low